packages/replay-internal/src/util/logger.ts

```diff
@@ -70,7 +70,7 @@ function makeReplayLogger(): ReplayLogger {
     });
 
     _logger.exception = (error: unknown, ...message: unknown[]) => {
-      if (_logger.error) {
+      if (message && _logger.error) {
```

Comment:
I think message will never be undefined, but an empty array? so this should probably be:

```suggestion
      if (message.length && _logger.error) {
```

🤔 ?

Reply:
Ah good catch, added some tests :)